core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -80,12 +80,16 @@ public static void main(String[] args) {
       LOG.info(USAGE);
       System.exit(-1);
     }
+    Mode mode = null;
     try {
-      format(Mode.valueOf(args[0].toUpperCase()));
+      mode = Mode.valueOf(args[0].toUpperCase());
     } catch (IllegalArgumentException e) {
       LOG.error("Unrecognized format mode: {}", args[0]);
       LOG.error("Usage: {}", USAGE);
       System.exit(-1);
+    }
+    try {
+      format(mode);
```

Comment:
IMO we shouldn't need to hardcode more information based on the exception path and the exception message itself should contain the necessary info. (ie. we could just do `LOG.error("Failed to format: {}, e.getMessage()")`)